api/firstbook.py

```diff
@@ -109,7 +110,8 @@ def request(self, url):
 
         Defined solely so it can be overridden in the mock.
         """
-        return requests.get(url)
```

Comment:
If we're constructing the auth header here this is no longer defined solely for that purpose!